src/tools/build-manifest/src/main.rs

```diff
@@ -473,6 +474,7 @@ impl Builder {
         // and so is rust-mingw if it's available for the target.
         components.extend(vec![
             host_component("rustc"),
+            host_component("rustc-dev"),
```

Comment:
I noticed that `rustc-dev` was automatically included when installing the dev-static try build, and I think it's because adding it to `components` here made it mandatory. That's fine for nightly, perhaps even what we want (for now), but we probably don't want that by default on beta or stable channels. I will add a conditional for this.

Reply:
Done.